netdisco/discoverables/apple_tv.py

```diff
@@ -13,5 +14,6 @@ def __init__(self, nd):
     def info_from_entry(self, entry):
         """Returns most important info from mDNS entries."""
         info = super().info_from_entry(entry)
-        info[ATTR_NAME] = info[ATTR_PROPERTIES]['name'].replace('\xa0', ' ')
+        info[ATTR_HOST] = str(ipaddress.ip_address(entry.address))
```

Comment:
Can you pick another attribute then ATTR_HOST? ATTR_HOST is standardized for MDNS discoverables: https://github.com/home-assistant/netdisco/blob/master/netdisco/discoverables/__init__.py#L121

Unless your implementation is better, in that case we might want to consider always returning `entry.address`